docs/sanity-migration/03-core-data-models.md

```diff
@@ -98,6 +98,63 @@ Show ─────────────────────────
 
 ---
 
+## Import Scripts Status
+
+### Current Import Scripts
+
+The following import scripts exist in `bin/migrations/` and can be run via npm:
+
+| Model | Import Script | npm Command |
+|-------|--------------|-------------|
+| Person/DJ | `importDeejays.ts` | `npm run import:deejays` |
+| Ad | `importAds.ts` | `npm run import:ads` |
+| Concert | `importConcerts.ts` | `npm run import:concerts` |
+| CdOfTheWeek | `importCdOfTheWeek.ts` | `npm run import:cdotw` |
+
+### Models That Still Need Import Scripts
+
+| Model | Schema Status | Import Status | Notes |
+|-------|--------------|---------------|-------|
+| Artist | ✅ Schema exists | ❌ No import script | May be created on-the-fly by Concert import |
+| Venue | ✅ Schema exists | ❌ No import script | May be created on-the-fly by Concert import |
+| Song (Music) | ✅ Schema exists | ❌ No import script | Needs `importMusic.ts` to migrate legacy `music` table with `featureOnNewMusic: true` |
+| Record | ✅ Schema exists | ❌ No import script | May need import if legacy records exist |
+| OnDemand | ❌ Schema needed | ❌ No import script | Full implementation needed |
+| Show | ❌ Schema needed | ❌ No import script | Full implementation needed |
+
+### Open Questions About Imports
+
+#### Artist Import
+1. **Do we need a standalone Artist import?** The Concert import creates Artists on-the-fly when they don't exist. Is there a separate legacy `artists` or `bands` table that should be migrated first?
+2. **Artist deduplication**: How do we handle variations in artist names (e.g., "The National" vs "National")?
+3. **Artist photos/bios**: Where do artist photos and bios come from in the legacy data? Concert records have `band_pic_url` but this is concert-specific.
+
+#### Venue Import
+1. **Do we need a standalone Venue import?** Similar to Artist, the Concert import creates Venues on-the-fly. Is there a separate legacy `venues` table?
```

Comment:
No standalone venue import needed. You're right they should be created on the fly.